src/RocketPoolEthTrigger.sol

```diff
@@ -0,0 +1,133 @@
+  /// price is deemed stale.
+  constructor(
+    IManager _manager,
+    AggregatorV3Interface _chainlinkOracle,
+    IRocketPoolOVMPriceOracle _rocketPoolOracle,
+    uint256 _priceTolerance,
+    uint256 _chainlinkFrequencyTolerance,
+    uint256 _rocketPoolFrequencyTolerance
+  ) BaseTrigger(_manager) {
+    if (_priceTolerance >= ZOC) revert InvalidPriceTolerance();
+    chainlinkOracle = _chainlinkOracle;
+    rocketPoolOracle = _rocketPoolOracle;
+    priceTolerance = _priceTolerance;
+    chainlinkFrequencyTolerance = _chainlinkFrequencyTolerance;
+    rocketPoolFrequencyTolerance = _rocketPoolFrequencyTolerance;
+
+    if (chainlinkOracle.decimals() != WAD_DECIMALS) revert InvalidDecimals();
+
+    runProgrammaticCheck();
+  }
+
+  /// @notice Compares the oracle's price to the reference oracle and toggles the trigger if required.
+  /// @dev This method executes the `programmaticCheck()` and makes the
+  /// required state changes both in the trigger and the sets.
+  function runProgrammaticCheck() public returns (MarketState) {
+    // Rather than revert if not active, we simply return the state and exit.
+    // Both behaviors are acceptable, but returning is friendlier to the caller
+    // as they don't need to handle a revert and can simply parse the
+    // transaction's logs to know if the call resulted in a state change.
+    if (state != MarketState.ACTIVE) return state;
+    if (programmaticCheck()) return _updateTriggerState(MarketState.TRIGGERED);
+    return state;
+  }
+
+  /// @notice Returns true if the trigger has been acknowledged by the entity responsible for transitioning trigger
+  /// state.
+  /// @notice Chainlink triggers are programmatic, so this always returns true.
+  function acknowledged() public pure override returns (bool) {
+    return true;
+  }
+
+  /// @dev Executes logic to programmatically determine if the trigger should be toggled.
+  function programmaticCheck() internal view returns (bool) {
+    (, int256 _chainlinkPriceInt,, uint256 _chainlinkUpdatedAt,) = chainlinkOracle.latestRoundData();
+    if (_chainlinkUpdatedAt > block.timestamp) revert InvalidTimestamp();
+    if (block.timestamp - _chainlinkUpdatedAt > chainlinkFrequencyTolerance) revert StaleOraclePrice();
+    if (_chainlinkPriceInt < 0) revert InvalidPrice();
+    uint256 _chainlinkPrice = uint256(_chainlinkPriceInt);
+
+    uint256 _rocketPoolPrice = rocketPoolOracle.rate();
+    uint256 _rocketPoolUpdatedAt = rocketPoolOracle.lastUpdated();
+    if (_rocketPoolUpdatedAt > block.timestamp) revert InvalidTimestamp();
+    // If things work as expected, the only delay should be from waiting for transaction inclusion in blocks every 5760 L1 blocks.
+    // If something goes wrong with Rocket Pool oracles, it might be delayed hours or days until software or nodes can be fixed.
+    if (block.timestamp - _rocketPoolUpdatedAt > rocketPoolFrequencyTolerance) revert StaleOraclePrice();
```

Comment:
I'm unsure whether we should include this, and if so, how long we would recommend setting `rocketPoolFrequencyTolerance` to..

5760 L1 blocks ~= 19.2 hours = 69120 seconds

I think we may want to recommend some buffer in case of waiting for transaction inclusion?